tests/tests/test_host_kdump.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/bash
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+if [ ! -v DEVICE_IP ]; then
+    DEVICE_IP="192.168.102.125"
+fi
+
+echo "Installing linux-crashdump and rebooting"
+ssh ubuntu@$DEVICE_IP sudo apt install linux-crashdump
+if [ $? != 0 ]; then
+    echo "Can't install linux-crashdump"
+    exit -1
+fi
+
+ssh ubuntu@$DEVICE_IP sudo systemctl reboot
+if [ $? != 0 ]; then
+    echo "Can't reboot"
+    exit -2
+fi
+
+sleep 30
+echo "Waiting for system to come back up"
+
+cnt=0
+until ssh ubuntu@$DEVICE_IP ls &> /dev/null; do sleep 1; cnt=$(expr $cnt + 1); if [ $cnt -gt 120 ]; then break; fi; done
```

Comment:
I have a few questions about this block:

1. Why increment to 120 but then check against 60 in the first if statement below? Shouldn't the two numbers match? Or can we just exit from the if statement inside the `until` block and then remove the `if [ $cnt -gt 60 ]` check below?
2. If I'm reading this correctly, we expect after the reboot that ssh-server will be up on the TDX host within 30 - 90 seconds (I guess that assumes that ssh returns immediately and does not hang). Is that actually a specific performance requirement some place?

Reply:
No performance requirement, just felt right.  The 120/60 thing was a mistake on my part.  I thought 120 was better.  In general, I don't want to fail if it just needs a little longer to boot.  Good find.  I'll fix the 120/60 thing.

Reply:
https://github.com/canonical/tdx/pull/237/commits/49834af5bd0588025ff659bb50199732297263c7